plugins/Toolbox/resources/qml/dialogs/ToolboxLicenseDialog.qml

```diff
@@ -20,31 +21,63 @@ UM.Dialog
     minimumHeight: UM.Theme.getSize("license_window_minimum").height
     width: minimumWidth
     height: minimumHeight
+    backgroundColor: UM.Theme.getColor("main_background")
+    margin: screenScaleFactor * 10
 
-    Item
+    ColumnLayout
     {
         anchors.fill: parent
+        spacing: UM.Theme.getSize("thick_margin").height
 
         UM.I18nCatalog{id: catalog; name: "cura"}
 
-
         Label
         {
             id: licenseHeader
-            anchors.top: parent.top
-            anchors.left: parent.left
-            anchors.right: parent.right
-            text: licenseModel.headerText
+            Layout.fillWidth: true
+            text: catalog.i18nc("@label", "You need to accept the license to install the package")
             wrapMode: Text.Wrap
             renderType: Text.NativeRendering
         }
+
+        Row {
+            id: packageRow
+
+            anchors.left: parent.left
+            anchors.right: parent.right
+            height: childrenRect.height
+            spacing: UM.Theme.getSize("default_margin").width
+            leftPadding: UM.Theme.getSize("narrow_margin").width
+
+            Image
+            {
+                id: icon
+                width: 30 * screenScaleFactor
+                height: width
+                fillMode: Image.PreserveAspectFit
+                source: licenseModel.iconUrl || "../../images/logobot.svg"
+                mipmap: true
+            }
+
+            Label
+            {
+                id: packageName
+                text: licenseModel.packageName
```

Comment:
This should be changed into the display name to be consistent with the previous page.